src/align_var_def_brace.cpp

```diff
@@ -113,20 +113,45 @@ Chunk *align_var_def_brace(Chunk *start, size_t span, size_t *p_nl_count)
    log_rule_B("align_mix_var_proto");
    bool  fp_active = options::align_mix_var_proto();
    Chunk *pc       = start->GetNext();
+   LOG_FMT(LAVDB, "%s(%d): start->Text() is '%s', level is %zu, brace_level is %zu\n",
+           __func__, __LINE__, start->IsNewline() ? "Newline" : start->Text(), start->level, start->brace_level);
 
-   while (  pc->IsNotNullChunk()
-         && (  pc->level >= start->level
-            || pc->level == 0))
+   while (pc->IsNotNullChunk())
    {
-      if (pc->IsNewline())
+      if (pc->Is(CT_NEWLINE))
+      {
+         LOG_FMT(LAVDB, "%s(%d): orig line is %zu, orig col is %zu, <Newline>, PRE is %s\n",
+                 __func__, __LINE__, pc->GetOrigLine(), pc->GetOrigCol(), pc->IsPreproc() ? "true" : "false");
+      }
+      else if (pc->Is(CT_NL_CONT))
       {
-         LOG_FMT(LAVDB, "%s(%d): orig line is %zu, orig col is %zu, <Newline>\n",
-                 __func__, __LINE__, pc->GetOrigLine(), pc->GetOrigCol());
+         LOG_FMT(LAVDB, "%s(%d): orig line is %zu, orig col is %zu, Text() '%s', type is %s, PRE is %s\n",
+                 __func__, __LINE__, pc->GetOrigLine(), pc->GetOrigCol(), pc->Text(), get_token_name(pc->GetType()), pc->IsPreproc() ? "true" : "false");
       }
       else
       {
-         LOG_FMT(LAVDB, "%s(%d): orig line is %zu, orig col is %zu, Text() '%s', type is %s\n",
-                 __func__, __LINE__, pc->GetOrigLine(), pc->GetOrigCol(), pc->Text(), get_token_name(pc->GetType()));
+         LOG_FMT(LAVDB, "%s(%d): orig line is %zu, orig col is %zu, Text() '%s', type is %s, PRE is %s\n",
+                 __func__, __LINE__, pc->GetOrigLine(), pc->GetOrigCol(), pc->Text(), get_token_name(pc->GetType()), pc->IsPreproc() ? "true" : "false");
+      }
+
+      if (  pc->level >= start->level
+         || pc->level == 0)
+      {
+         // go on
+      }
+      else
+      {
+         //
+         if (pc->IsPreproc())
+         {
+            // go on
+         }
+         else
+         {
+            LOG_FMT(LAVDB, "%s(%d): orig line is %zu, orig col is %zu, Text() '%s', type is %s, PRE is %s\n",
+                    __func__, __LINE__, pc->GetOrigLine(), pc->GetOrigCol(), pc->Text(), get_token_name(pc->GetType()), pc->IsPreproc() ? "true" : "false");
+            break;
+         }
```

Comment:
This could be simplified in
```
      if (  pc->level >= start->level
         || pc->level == 0
         || pc->IsPreproc())
      {
         // go on
      }
      else
      {
         LOG_FMT(...
         break;
      }
```
or even better
```
      if (  pc->level < start->level
         && pc->level != 0
         && !pc->IsPreproc())
      {
         LOG_FMT(...
         break;
      }
```